### PR TITLE
Use Qt 5.12

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: defaults.sh [hunk 1]
```diff
@@ -103,12 +103,12 @@ BUILD_TOOLS_SRC=$(cd "$(dirname "$0")" && pwd)
...
 
 # Source directory
-DEF_QT_SOURCE_PACKAGE=qt-everywhere-opensource-src-$DEF_QT_VER
+DEF_QT_SOURCE_PACKAGE=qt-everywhere-src-$DEF_QT_VER
 DEF_QT_SRC_DIR=$DEF_PREFIX/invariant/$DEF_QT_SOURCE_PACKAGE
 
 if [[ $UNAME_SYSTEM == "Linux" ]] || [[ $UNAME_SYSTEM == "Darwin" ]]; then
     # Dynamic Qt build directory on Linux and MacOS
-    DEF_QT_DYN_BUILD_DIR=$DEF_PREFIX/invariant/$DEF_QT_SOURCE_PACKAGE-build
+    DEF_QT_DYN_BUILD_DIR=$DEF_PREFIX/invariant/$DEF_QT_SOURCE_PACKAGE-build-rpathpadrpathpad
```
Comment (vigejolla) [inline defaults.sh]: rpathpadrpathpad?

Comment (martyone) [reply defaults.sh]: lovely, right? :)

Comment (vigejolla) [reply defaults.sh]: It sure is, but what's the reason behind that?

Comment (martyone) [reply defaults.sh]: RPATH is used to resolve library dependencies under user provided installation dir. RPATH is modified during Qt Creator's make-install phase when Qt is packaged. New RPATH needs to fit into the space occupied by the original RPATH. Hence the padding to reserve more space. The original name contained the word "-opensource" which did the trick. Similar issue arises every now and then.

rpath-pad - self documenting :)

Comment (vigejolla) [reply defaults.sh]: I was quite close with my guesses then. I was just a bit confused why we didn't need it before.

**Pull request merged**